labels.py

```diff
@@ -0,0 +1,26 @@
+from cProfile import label
+from lib2to3.pytree import LeafPattern
+from Labelling.config.constants import TEST_IMG_DIR_PATH
+
+import shutil
+import os
+
+
+LABEL_DIR = TEST_IMG_DIR_PATH
+NEW_PATH = os.path.abspath('Labels')
+
+print(NEW_PATH)
+
+for path, dirs, files in os.walk(LABEL_DIR):
+    for file in files:
+        if file.endswith('json'):
```

Comment:
Maybe use `if file.endswith('.json'):` just to eliminate the possibility of a file called `json` being included here
***
Side note: There is an os command for this, [here](https://docs.python.org/3/library/os.path.html#os.path.splitext)

I admit it may be less elegant, but I think it's more rigourous (unless the .json is `.......json`, [see caveats of the function](https://docs.python.org/3/library/os.path.html#os.path.splitext)... so maybe the original is more rigourous!)

```python
        if os.path.splitext(file)[-1] == ".json":
            ....
```

Reply:
Another note: make sure this takes into account the case where someone uses `.JSON` files (or `.JsON` or `.jSoN`... I don't know why people would do this lol). The python string method `.lower()` might be useful here.